simpleflow/task.py

```diff
@@ -49,6 +49,12 @@ class ActivityTask(Task):
     def __init__(self, activity, *args, **kwargs):
         if not isinstance(activity, Activity):
             raise TypeError('Wrong value for `activity`, got {} instead'.format(type(activity)))
+        # Keep original arguments for use in subclasses
+        # For instance this helps casting a generic class to a simpleflow.swf.task,
+        # see simpleflow.swf.task.ActivityTask.from_generic_task() factory
+        self._args = args
+        self._kwargs = kwargs
```

Comment:
Yes... Either `deepcopy` or add to `from_generic_task` with a comment, as you want